FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -78,7 +82,7 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 .GetRequiredService<IOptions<SchemaConfiguration>>()
                 .Value;
 
-            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.CustomizedSchemaImageReference))
+            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.ContainerRegistry.SchemaImageReference))
```

Comment:
And should we check whether the `SchemaImageReference` is in valid ACR image format? Or it will be checked in another place, or our strategy here is "wait-for-exception-and-wrap-throw"?